src/explore-education-statistics-common/src/modules/charts/util/__tests__/getMinorAxisSize.test.ts

```diff
@@ -0,0 +1,71 @@
+import getMinorAxisSize from '@common/modules/charts/util/getMinorAxisSize';
+
+describe('getMinorAxisSize', () => {
+  const testDataSetCategories = [
```

Comment:
Would add explicit type as untyped variables can easily get out of sync with the reality - particularly in tests.